3-things-you-must-know-about-midnight-hacks

```diff
@@ -0,0 +1,45 @@
+{  
+  "author": "Kara Dees",
+  "title": "3 Things You Must Know About Midnight Hacks",
+  "thumbnail": "http://imgur.com/a/YATmL",
+  "date": "2017-03-03T02:15:35Z",
+  "tags": ["all"],
+  "published": true
+}
+
+---
+
+##1.  It’s a Phenomenon##
+
+What comes to mind when you hear the phrase “Overnight Social Hacking”?  These three words are the basis of a new phenomenon on Georgia State’s campus. Midnight Hacks, a monthly event put on by Panther Hackers (PH), was last held on February 24th, 2017 in the new LaunchGSU space. It was wildly successful through the combination of caffeine, coding, and gaming into one 8-hour, overnight experience for students.
+
+Phillip Kinney, sophomore computer science major, was the Event Manager. When recounting the event he said, “The appeal to an overnight event like Midnight Hacks is to be able to collaborate on projects late at night and to do Smash Brothers ‘til the sun comes up.”
+
+</br>
+</br>
+##2. It’s a Party With You and 30+ of Your Closest Friends##
+
+PH is an inclusive community. New people are always welcome. Things like the Smash Brothers tournament are oriented towards students who want to meet others in a low stress atmosphere.
+
+Imanuel Sonubi, freshman computer science major, when asked about his experience at Midnight Hacks said, “I had a lot of fun just chilling with members of PH who are like family at this point. I learned about an indie game called 20xx, and also played Dungeon Souls & Smash Brothers.”
+
+![alt text](http://imgur.com/a/o3tc8)
```

Comment:
The `[alt text]` means "alternative text." It will show up when the picture can't load. It will also be used by accessibility features, like screen readers for blind people. You should change it for something that describes the picture :)